spot_rl_experiments/spot_rl/envs/place_env.py

```diff
@@ -2,69 +2,235 @@
+
+        Returns:
+            ee_pos: Gripper position as an np.array([x,y,z])
+        """
+        # Get end effector position (in base frame) .. (x,z,-y) -> (x,-y,z)
+        ee_pos = np.array(self.place_env.get_in_gripper_tf().translation)[
+            np.array([0, 2, 1])
+        ]
+        # Correct the sign on "y"
+        ee_pos[1] *= -1
+        return ee_pos
+
+    # Should this be kept here or moved to spot.py?????
+    def get_place_target_in_base_frame(self):
+        """
+        Returns the current place target in the base frame as (x,y,z)
+
+        Returns:
+            base_frame_place_target_corrected: Place target in base frame as an np.array([x,y,z])
+        """
+        # Get place target (in base frame) .. x,z,-y -> (x,-y,z)
+        base_frame_place_target_uncorrected = (
+            self.place_env.get_base_frame_place_target()
+        )
+        base_frame_place_target_corrected = np.array(
+            self.place_env.spot2habitat_translation(base_frame_place_target_uncorrected)
+        )[np.array([0, 2, 1])]
+
+        # Correct the sign on "y"
+        base_frame_place_target_corrected[1] *= -1
+        return base_frame_place_target_corrected
+
+    def shutdown(self, should_dock=False):
+        try:
+            if should_dock:
+                # self.place_env.say("Executing automatic docking")
+                dock_start_time = time.time()
+                while time.time() - dock_start_time < 2:
+                    try:
+                        self.spot.dock(dock_id=DOCK_ID, home_robot=True)
+                    except Exception:
+                        print("Dock not found... trying again")
+                        time.sleep(0.1)
+            else:
+                # self.place_env.say("Will sit down here")
+                self.spot.sit()
+        finally:
+            self.spot.power_off()
 
 
 class SpotPlaceEnv(SpotBaseEnv):
-    def __init__(self, config, spot: Spot, place_target, target_is_local=False):
+    def __init__(self, config, spot: Spot):
         super().__init__(config, spot)
-        self.place_target = np.array(place_target)
-        self.place_target_is_local = target_is_local
+        self.place_target = None
+        self.place_target_is_local = False
+
         self.ee_gripper_offset = mn.Vector3(config.EE_GRIPPER_OFFSET)
         self.placed = False
 
-    def reset(self, *args, **kwargs):
+    def reset(self, place_target, target_is_local=False, *args, **kwargs):
+        # assert place_target is not None
+        self.place_target = np.array(place_target)
+        self.place_target_is_local = target_is_local
+
         # Move arm to initial configuration
         cmd_id = self.spot.set_arm_joint_positions(
             positions=self.initial_arm_joint_angles, travel_time=0.75
         )
         self.spot.block_until_arm_arrives(cmd_id, timeout_sec=2)
 
-        observations = super(SpotPlaceEnv, self).reset()
```

Comment:
This seemed like a redundant `super()` call. So updated it to make it similar to how other Envs are calling it.
FYI - @jimmytyyang @akshararai